sql/src/main/java/org/apache/druid/sql/calcite/rel/Windowing.java

```diff
@@ -270,6 +279,74 @@ public static Windowing fromCalciteStuff(
     }
   }
 
+  /**
+   * A wrapper class which stores {@link WindowGroup}
+   * along with its computed {@link WindowOperatorFactory}
+   * <p>
+   * this allows us to sort the window groups in order to optimise the order of operators we would need to compute
+   * without losing the aggregate column name information (which is part of the computed WindowOperatorFactory)
+   */
```

Comment:
I don't feel like this apidoc adds much detail - I believe it would be ok even without it.

Usually I think its better to have the apidoc describe the role or mission of the class - I believe that adding classes like this are part of an emerging architecture.